src/main/java/by/vadzimmatsiushonak/bank/api/controller/BankPaymentController.java

```diff
@@ -73,10 +83,24 @@ public ResponseEntity<BankPaymentDto> create(
             @ApiResponse(code = HTTP_BAD_REQUEST, message = "Invalid arguments provided")})
     @ResponseStatus(CREATED)
     @PostMapping("/initiatePayment")
-    public ResponseEntity<BankPaymentStatusResponse> initiatePayment(
+    public ResponseEntity<BankPaymentConfirmationResponse> initiatePayment(
```

Comment:
it's better to rename "BankPaymentConfirmationResponse" with "BankPaymentResponse"